Create kmc model

meelod opened the pull request:
- Added create_kmc_model() function to return Project object
Added save_model = save pointer

kmcos committed:
Merge pull request #136 from kmcos/develop

Develop

meelod committed:
create_kmc_model

- Added create_kmc_model() function to return Project object
Added save_model = save pointer

meelod commented:
Do not merge yet

AdityaSavara commented on kmcos/types.py:
```diff
@@ -81,8 +81,12 @@ class Project(object):
     """
 
-    def __init__(self):
+    def __init__(self, **name):
```
@meelod 
def __init__(self, **name): 
should be 
def __init__(self, model_name=None):

meelod committed:
create_kmc_model() & compile()

- Created default model name to project class
- create_kmc_model method creates a Project object that takes optional model_name argument
- compile() takes a kmc model from create_kmc_model and exports as xml and creates a local smart directory

meelod committed:
set_meta

- comparisons to variable types

AdityaSavara reviewed (approved):
We have tested some unit tests, and have more to test.
